src/momento/incubating/README.md

```diff
@@ -74,10 +76,69 @@ CacheDictionaryGetAllResponse(value={b'key1': b'value1', b'key2': b'value2'}, st
 {b'key1': b'value1', b'key2': b'value2'}
 ```
 
-7. Interact with a returned dictionary
+6. Interact with a returned dictionary
 
 ```python
 >>> my_dictionary = dictionary_get_all_response.value()
 >>> my_dictionary["key1"]
 'value1'
 ```
+
+## Exists
+
+This demonstrates the methods and response features for testing the existence of key(s).
+
+1. Test a single key
+
+```python
+>>> if client.exists("my-cache", "key"):
+... 	# Key exists
+... else:
+... 	# Handle missing keys
+```
+
+2. Test multiple keys
+
+```python
+>>> response = client.exists("my-cache", *keys)
+>>> if not response.all():
+... 	# At least one key is missing
+```
+
+3. Find number of keys that exist
+
+```python
+>>> response = client.exists("my-cache", *keys)
+>>> response.num_exists()
+```
+
+4. Find keys that did not exist
+
+```python
+>>> response = client.exists("my-cache", *keys)
+>>> response.missing_keys()
+```
+
+5. Find keys that exist
+
+```python
+>>> response = client.exists("my-cache", *keys)
+>>> response.present_keys()
+```
+
+6. Iterate over all keys
+
+```python
+>>> for key, does_exist in client.exists("my-cache", *keys).zip_keys_and_results():
```

Comment:
any reason not to just return a `dict` on this one?  callers can use language constructs to decide how they want to iterate over the results.

or maybe that is what you are returning here and python's `for` is just destructuring?  if that's the case, not sure you need the word `zip` in the fn name?

I would maybe just call this `results()` and have it return a `dict[key, bool]`.

tbh I'm not even sure if you need the `missing_keys` / `present_keys` with that approach since callers can just do a filter operation.  Those functions seem nice but i feel like it's always best to try to minimize the surface area of an API at the beginning and only add new things as people run into concrete requirements for them that can't be trivially solved in the language itself.